doc/remailers.rst

```diff
@@ -0,0 +1,125 @@
+Autocrypt and Mailing Lists
+====
+
+General Considerations
+----
+
+"Mailing lists" are a special use case of e-mail, where some address is managed
+not as an inbox that stores e-mail for retrieval, but instead forwards all
+messages to a configurable list of subscribers. For the case of end-to-end
+encryption, the introduction of a relaying agent brings up the question of what
+constitutes an "Endpoint".
+
+For users with a typical intuition of end-to-end encryption, the expected end
+points of end-to-end encrypted mailing lists will likely be the individual
+subscribers. This is very difficult to achieve on top of existing e-mail
+infrastructure, since it requires either negotiation between all endpoints, or
```

Comment:
"negotiation between all endpoints" should say something like: e.g., an explicitly Cc'ed list of recipients, with everyone doing "reply all".

Reply:
Hm. My thought was that mailing lists can still be used for delivery, it's just the e2e crypto that requires negotiation between all endpoints. A Cc'ed list of recipients should be distinctly *not* difficult to achieve, given `Autocrypt-Gossip` :) should make that point clearer